src/commands/reputation/giveReputation.test.ts

```diff
@@ -99,7 +75,7 @@ describe('giveRep', () => {
     expect(replyMock).not.toHaveBeenCalled();
   });
 
-  it('should send reject message if user mention himself', async () => {
+  it('should send reject message if user mention themselves', async () => {
```

Comment:
I think we don't need to mention multiple mentions here, hence `themself` is enough.

```suggestion
  it('should send reject message if user mention themself', async () => {
```